src/routes/blog/spyder-6-remote-development/+page.md

```diff
@@ -0,0 +1,94 @@
+---
+title: The inside scoop on Spyder 6's new remote development platform
+author:
+  - hlouzada
+  - ccordoba12
```

Comment:
```suggestion
```

I don't think I should be a co-author in this one (my writing contribution is fairly small).